### DISCUSSION TITLE
Why do `@pnpm/*` packages include themselves in `devDependencies`?

### DISCUSSION THREAD

**Discussion by andy-zhou** [pnpm/pnpm#7629] [General]
I was looking through `.meta-updater` to help manage my own monorepo and I noticed these lines:

https://github.com/pnpm/pnpm/blob/3c7c0926f68f918d573fa2d8d75f570497bea46e/.meta-updater/src/index.ts#L34-L37

It looks like every package adds itself as a `devDependency`:

https://github.com/pnpm/pnpm/blob/3c7c0926f68f918d573fa2d8d75f570497bea46e/packages/git-utils/package.json#L38-L41

What's the rationale behind this? Is this something we should be considering in our own monorepos?

**Comment by zkochan** [comment]
it just makes it easier to require the package from the tests. So in the tests of `@pnpm/core` we can import things from core via `import {} from '@pnpm/core'` instead of `import {} from '../../dist'`
